### DISCUSSION TITLE
Tooltip show ONLY if ellipsis present?

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#9007] [General - Components / Directives / etc]
Is there a way to have a q-tooltip show on hover only if the (in this case table cell) I am using it in has had to truncate its content via css ellipsis? Ideally if the entire text of the cell is showing there is no need for it (or if the cell is blank for that matter).

**Comment by metalsadman** [comment]
set a `@mouseenter` if you want it on hover per cell or use some class that you can collect on mounted via document selector and check for element's `el.offsetWidth < el.scrollWidth` if true then set a state or something that is used by your tooltip, ie. `v-if="props.row.isShowTooltip"` or etc...

ref. https://stackoverflow.com/a/56540371
